sw/deployer/src/main.rs

```diff
@@ -272,6 +272,14 @@ enum SDCommands {
     /// Format the SD card
     #[command(name = "mkfs")]
     Format,
+
+    /// Mount the SD card
+    #[command(name = "fuse")]
+    #[cfg(unix)]
+    Fuse {
+        /// Path to the directory
+        mount_path: PathBuf,
```

Comment:
Change the name of parameter to `path` and explain what it does in the description one line above. "Path to the directory" tells noting in this context.